packages/@cdktf/hcl2cdk/package.json

```diff
@@ -31,11 +31,11 @@
     "@babel/template": "^7.14.5",
     "@babel/types": "^7.14.5",
     "@cdktf/hcl2json": "*",
+    "@cdktf/provider-generator": "*",
```

Comment:
I know this change is not new, but other packages depend on `0.0.0` instead of `*` and I think that's then replaced by the current version somehow.

Reply:
Yeah sounds good